frontend/src/app/components/project-dashboard/project-dashboard.component.html

```diff
@@ -110,18 +110,7 @@ <h3 class="mat-title" *ngIf="project">Boards for {{ project.name }}</h3>
                 <mat-card-title class="title accent">
                   <mat-icon color="accent">picture_in_picture</mat-icon>
                   <span>{{ board.name }}</span>
-                </mat-card-title>
-                <button
-                  mat-icon-button
-                  class="accent"
-                  *ngIf="user && user.role === Role.TEACHER"
-                  matTooltip="Toggle Visibility"
-                  (click)="toggleBoardVisibility($event, board)"
-                >
-                  <mat-icon *ngIf="board.visible">visibility</mat-icon>
-                  <mat-icon *ngIf="!board.visible">visibility_off</mat-icon>
-                </button>
-                <button 
+                  <button 
```

Comment:
Amazing! You made quick work of this. Just adjust the indentation here, then looks good to me. Let's wait for this commit and Marie's review then we can merge.

Looks like I'll need to find something more challenging for you!